### ISSUE TITLE
High Contrast-White: Focus not visible on Dropdown in HC white mode

### ISSUE THREAD

**Issue by harshil93** [OfficeDev/office-ui-fabric-react#1673]
Pre-requisite:
Enable high contrast
Press left Shift+left Alt+Print Screen (or) select the "Ease of Access"  button in the lower-right corner and choose High Contrast (White) and then hit apply. 

Observe the Focus on dropdown when selected. 
It can be reproduced here : https://fabricreact.azurewebsites.net/fabric-react/master/

Expected Behavior:
Clear focus should be visible on the selected control

User Impact:
Low vision users using high contrast themes will not be able to differentiate between the buttons and links

Affected Branch: Windows 10 RS1
Affected Build: 14393.935
Browser Version: Microsoft Edge 38.14393.0.0

**Comment by stephenzsy** [comment]
We are blocked on the same issue.

**Comment by mdahamiwal** [comment]
Hi @harshil93, @stephenzsy, we did high contrast improvements in dropdown with https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/office-ui-fabric-react_v2.25.1

Here is how it looks in HC white now. Let us know if this is still an issue:

![dropdown-white](https://cloud.githubusercontent.com/assets/12109362/25837575/1cd62a50-34ac-11e7-93ae-95f224f44c05.gif)

**Comment by betrue-final-final** [comment]
This looks great!

**Comment by betrue-final-final** [comment]
What does it look like in high contrast black?
